yantra/manager.py

```diff
@@ -45,21 +45,22 @@ def _get_modules(self):
         """Discover all modules in the path"""
         plugin_modules = []
 
-        # walk the plugins folder
-        for root, subdir, files in os.walk(self.plugin_type.path):
+        for p in self.plugin_type.paths:
+            # walk the plugins folder
+            for root, subdir, files in os.walk(p):
```

Comment:
This method is getting a bit ridiculous at this point. Might be good to extract the walking part into another method that takes a plugin.